src/objects/zcl_abapgit_objects.clas.abap

```diff
@@ -888,13 +897,35 @@ CLASS zcl_abapgit_objects IMPLEMENTATION.
 
   METHOD is_type_supported.
 
-    " If necessary, initialize list
-    IF gt_supported_obj_types IS INITIAL.
-      supported_list( ).
+    DATA: ls_item               TYPE zif_abapgit_definitions=>ty_item,
+          ls_supported_obj_type TYPE ty_supported_types.
+
+    FIELD-SYMBOLS <ls_supported_obj_type> TYPE ty_supported_types.
+
+    IF iv_obj_type IS INITIAL.
+      " empty object type should never exist
+      RETURN.
+    ENDIF.
+
+    READ TABLE gt_supported_obj_types
+      ASSIGNING <ls_supported_obj_type>
+      WITH KEY obj_type = iv_obj_type.
```

Comment:
@manhofmann You should use at this read table with with **table** key instead of with key, because gt_supported_obj_types is a sorted table with the unique key obj_type. So with table key is faster.